muse/CONST.py

```diff
@@ -1,6 +1,7 @@
 import os
 import uuid
 
+ENABLE_TRACKERS = os.environ.get("ENABLE_TRACKERS", False)
```

Comment:
```suggestion
ENABLE_TRACKERS = os.environ.get("MUSE_ENABLE_TRACKERS", False)
```
@yurijmikhalevich we have a practice of adding a prefix of the project so that it's interfere with user's env variblaes